cancermuts/datasources.py

```diff
@@ -334,10 +335,13 @@ def cancer_studies(self):
     def add_mutations(self, sequence, metadata=[]):
         _cBioPortal_supported_metadata = ['cancer_type', 'cancer_study', 'genomic_coordinates', 'genomic_mutations']
 
+        if not sequence.is_canonical:
+            raise UnexpectedIsoformError("cBioPortal mutation annotation only supports canonical isoforms. Please use a canonical UniProt sequence")
```

Comment:
I'd replace the last sentence with "Please use a Sequence object for a canonical isoform"